highlight/highlight
# Fix expand link in related resource panel

## Summary

Fixes a bug where clicking the expand button in the related resource panel on a trace from the traces page wouldn't work. Also updates to use a `Link` instead of of `navigate` and an onClick handler which allows you to command + click or right click to open in a new tab.

## How did you test this change?

* Open a trace from the traces page
* Click the expand button - nothing should happen :)
* View other related resources across the app and click the expand button to ensure they work as expected

## Are there any deployment considerations?

N/A - client-side change only.

## Does this work require review from our design team?

N/A - no change in UI.
